src/main/java/io/kabanero/api/GithubClientHelper.java

```diff
@@ -0,0 +1,24 @@
+package io.kabanero.api;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import io.kabanero.v1alpha2.client.apis.KabaneroApi;
+import io.kabanero.v1alpha2.models.Kabanero;
+import io.kubernetes.KabaneroClient;
+import io.kubernetes.client.ApiClient;
+
+public class GithubClientHelper {
```

Comment:
this class should also have a method to getGitHubClient() which will return a GitHubClient class that is pre configured.

`getApiURL` seems like it could be a good **private** helper method in this class

Reply:
also maybe instead of "GitHubClientHelper" as the class name we should consider "GitHubClientInitilizer" since Helper doesn't say much about the class and initializer sounds more descriptive.